sklearn/decomposition/tests/test_fastica.py

```diff
@@ -46,6 +45,10 @@ def test_gs():
     assert (tmp[:5] ** 2).sum() < 1.0e-10
 
 
+# FIXME remove filter in 1.3
+@pytest.mark.filterwarnings(
+    "ignore:From version 1.3 whiten='unit-variance' will be used by default."
+)
```

Comment:
Is this really needed? The `whiten` parameter is passed explicitly in this test.

Reply:
It is needed because "arbitrary-variance" is tested.

Reply:
I would rather put a local ignore warning context manager where it's actually needed in the test but no big deal if you prefer to keep it this way.

Reply:
So would I, but there is already a context manager for `ValueErrors` on the call which warns.
Moreover, adding a context on top would just add a fourth indentation level. :/

I think it's fine to keep those 4 lines, which kind of looks like a convention in tests for temporarily filtering warnings.